exporter/opentelemetry-exporter-otlp-proto-grpc/src/opentelemetry/exporter/otlp/proto/grpc/trace_exporter/__init__.py

```diff
@@ -84,6 +85,9 @@ def __init__(
         timeout: Optional[int] = None,
         compression: Optional[Compression] = None,
     ):
+
+
+        insecure = insecure or environ.get(OTEL_EXPORTER_OTLP_INSECURE, False)
```

Comment:
Should we change the constructor default for `insecure` to `False` instead of `None`?